core/src/avm2/globals/flash/system/capabilities.rs

```diff
@@ -0,0 +1,33 @@
+//! `flash.display.Capabilities` native methods
+
+use crate::avm2::{Activation, AvmString, Error, Object, Value};
+
+/// Implements `flash.system.Capabilities.version`
+pub fn get_version<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    _this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    // TODO: Report the correct OS instead of always reporting Linux
```

Comment:
(I would guess that one could make an issue for fully implementing this, but, the application developer should not be relying on this either way (it even says so in the official documentation), so I do not believe adding an issue for this is necessary. There are stubs for the other methods as well).

Reply:
It's just a TODO notice to state that "this is not fully implemented yet" without marking it as a stub because the most important functionality is implemented.